src/modules/land_detector/land_detector_params_mc.c

```diff
@@ -84,3 +84,17 @@ PARAM_DEFINE_FLOAT(LNDMC_ROT_MAX, 20.0f);
  *
  */
 PARAM_DEFINE_FLOAT(LNDMC_ALT_MAX, -1.0f);
+
+/**
+ * Ground effect altitude for multicopters
+ *
+ * The height above ground below which ground effect creates barometric altitude errors.
+ * A negative value indicates no ground effect.
+ *
+ * @unit m
+ * @min -1
+ * @decimal 2
+ * @group Land Detector
+ *
+ */
+PARAM_DEFINE_FLOAT(LNDMC_ALT_GND, -1.0f);
```

Comment:
Thoughts on a conservative default value rather than disabled?

Reply:
Because the dead zone in the EKf is off by default, there is not point setting a positive default for LNDMC_ALT_GND this unless we also set a positive default for EKF2_GND_EFF_DZ. If I had to pick values it would be:

LNDMC_ALT_GND = 0.5
EKF2_GND_EFF_DZ = 5.0

however the amount of positive pressure/negative height disturbance due to rotor/ ground interaction which determines the optimum value EKF2_GND_EFF_DZ  is highly sensitive to autopilot placement and placement of openings in the hull, whereas LNDMC_ALT_GND tends to scale with the sqrt of rotor area.